src/clj/rems/api/schema.clj

```diff
@@ -51,6 +60,7 @@
 
 (s/defschema Entitlement
   {:resource s/Str
+   :userid UserId ;; TODO could have UserWithAttributes here instead of :userid and :mail
```

Comment:
👍

Reply:
For a transition period we could have both.

Reply:
hmm okay :userid is fresh so I can replace it with :user, and keep :mail with a deprecation comment